screens/Scan/SharingErrorModal.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {useTranslation} from 'react-i18next';
+import {Theme} from '../../components/ui/styleUtils';
+import {Modal} from '../../components/ui/Modal';
+import {Pressable, Dimensions} from 'react-native';
+import {Button, Column, Text} from '../../components/ui';
+import {useScanLayout} from './ScanLayoutController';
+import {useSendVcScreen} from './SendVcScreenController';
+import testIDProps from '../../shared/commonUtil';
+import {SvgImage} from '../../components/ui/svg';
+import {Icon} from 'react-native-elements';
+
+export const SharingErrorModal: React.FC<
+  FailureErrorOfVcShareProps
+> = props => {
+  const {t} = useTranslation('ScanScreen');
+
+  return (
```

Comment:
reg this - https://github.com/mosip/inji/pull/1210/files#r1471043167
  

`Existing error modal, successShareModal, SharingErrorModals are diff from each other in UI. Previous is related to network error.
All there are differ with buttons g their navigation.`

We have the similar pattern available in 
- Error component - error image, title, message , try again & goBack (goBack implemented in INJIMOB-691 PR), 
- if required as modal, we can use isModal prop there. 

Given this pattern is available, shall we try using existing error component and adjust based on needs? 
Please let me know if we have further complications here.